neurodocker/interfaces/minc.py

```diff
@@ -0,0 +1,128 @@
+                   "#--------------------".format(self.version))
+
+        if self.use_binaries:
+            chunks = [comment, self.install_binaries()]
+        else:
+            raise ValueError("`use_binaries=True` is the only available "
+                             "option at this time.")
+
+        return "\n".join(chunks)
+
+    def _get_binaries_urls(self, version):
+        try:
+            return MINC.VERSION_Releases[version]
+        except KeyError:
+            raise ValueError("MINC version not available: {}".format(version))
+
+    def _get_binaries_dependencies(self):
+        base_deps = {
+            'apt': 'libc6 libstdc++6 imagemagick perl unzip',
+            'yum': 'glibc libstdc++ ImageMagick perl unzip',
+        }
+        return base_deps[self.pkg_manager]
+
+    def _install_binaries_deps(self):
+        """Install the dependencies for binary installation
+        """
+        cmd = "{install}\n&& {clean}".format(**manage_pkgs[self.pkg_manager])
+        return cmd.format(pkgs=self._get_binaries_dependencies())
+
+    def _get_install_cmd(self, minc_url, beast_url, models_90a_url, models_90c_url, entrypoint_cmd):
+        cmd = ('echo " Downloading MINC, BEASTLIB, and MODELS..."'
+               "\n&& curl -sSL --retry 5 {minc_url}"
+               "\n| tar zx -C /opt"
+               "\n&& curl -sSL --retry 5 {beast_url}"
+               "\n| tar zx -C /opt/minc/share"
+               "\n&& curl -sSL --retry 5 -o /tmp/mni_90a.zip {models_09a_url}"
+               "\n&& unzip /tmp/mni_90a.zip -d /opt/minc/share/icbm152_model_09a"
+               "\n&& curl -sSL --retry 5 -o /tmp/mni_90c.zip {models_09c_url}"
+               "\n&& unzip /tmp/mni_90c.zip -d /opt/minc/share/icbm152_model_09c"
+               "\n&& rm -r /tmp/mni_90* "
+               "\n&& {entrypoint_cmd}".format(minc_url=minc_url, beast_url=beast_url, models_09a_url=models_90a_url,
+                                              models_09c_url=models_90c_url, entrypoint_cmd=entrypoint_cmd))
+        cmd = indent("RUN", cmd)
+        return cmd
+
+
+    def install_binaries(self):
+        """Return Dockerfile instructions to download and install MINC
+        binaries.
+        """
+        from neurodocker.dockerfile import _add_to_entrypoint
+        minc_url = self._get_binaries_urls(self.version)
+        beast_url = self.BEAST_URL['latest']
+        models_09a_url = self.MODELS_09a_URL['latest']
+        models_09c_url = self.MODELS_09c_URL['latest']
+        if self.check_urls:
+            check_url(minc_url)
+            check_url(beast_url)
+            check_url(models_09a_url)
+            check_url(models_09c_url)
+
+        deps_cmd = self._install_binaries_deps()
+        deps_cmd = indent("RUN", deps_cmd)
+
+        ent = _add_to_entrypoint("source /opt/minc/minc-toolkit-config.sh",
+                                 with_run=False)
+
+        cmd = self._get_install_cmd(minc_url, beast_url, models_09a_url, models_09c_url, ent)
```

Comment:
Can you put `deps_cmd` and `cmd` in the same `RUN` instruction? I don't think it is necessary to have them as separate layers.